vnext/PropertySheets/External/Microsoft.ReactNative.Uwp.CSharpLib.targets

```diff
@@ -10,24 +10,19 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Common.targets" />
 
+  <!-- Due to visual studio unconditionally showing references, we have to trick it by making it impossible for VS to find the reference differences between building as source and building as NuGet -->
```

Comment:
- Why not keep the name in line with the MSBuild construct (`ProjectReferences` instead of `SourceReferences`)?
- Why not use `props` instead of `targets` as an extension (there are no targets defined on that file)?